docs-parts/queries/03-Fetch_lang1.rst

```diff
@@ -35,6 +35,25 @@ Primary key values
 
 ``KEY`` can also used when returning attribute values as separate variables, such that one of the returned variables contains the entire primary keys.
 
+Sorting and limiting the results
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To sort the result, add the additional ``order_by`` argument to ``fetch`` .
+
+.. code-block:: python
+
+    data = query.fetch(order_by='name')
+
+The ``order_by`` argument is passed directly to SQL and follows the same syntax as the `ORDER BY clause <https://dev.mysql.com/doc/refman/5.7/en/order-by-optimization.html>`_
```

Comment:
`fetch` does a few things with the `order_by` argument and it is not directly passed to the SQL. For example, `order_by` can be a tuple.

Reply:
did not know that - should have cross-checked docs. thanks :)